### ISSUE TITLE
HDMI switching 

### ISSUE THREAD

**Issue by xadacka** [AlphamaxMedia/netv2-fpga#10]
Hey, I'm not sure if any aspect of this side of the hardware has been developed yet but I was wondering about the hdmi switch option that was briefly mentioned in the campaign, is it currently possible to connect a second source to the internal mini hdmi jack and switch between the inputs? Or, is it even for switching between source inputs or is it only for overlay inputs or is it open to any use?

I've no immediate use case for this functionality so I'm just enquiring more than anything to try think of what other uses I can come up with. Thanks!

**Comment by bunnie** [comment]
Yes, it is possible to connect a second source to the micro HDMI connector on the inside. The port is tested during manufacturing, so it should be fully functional.

However, someone would need to modify an NeTV2 FPGA design to utilize this feature.
